lib/react_client/react_interop.dart

```diff
@@ -34,8 +36,54 @@ abstract class React {
 
   external static bool isValidElement(dynamic object);
   external static ReactClass get Fragment;
+
+  /// Returns a ref that can be attached to an element via the ref attribute.
+  ///
+  /// When a ref is passed to an element in render, a reference to the node becomes
+  /// accessible at the 'current' attribute of the ref.
+  ///
+  /// See: <https://reactjs.org/docs/refs-and-the-dom.html#creating-refs>.
+  external static createRef();
 }
 
+class createRef<CurrentType> {
```

Comment:
Oh I guess we could just have a like function that returns a new class instance of this class, I was thinking since you need to construct it similarly to the function we wouldn’t need it but that makes sense. So something like
`CreateRef createRef() => new CreateRef();`
?